todo.js

```diff
@@ -0,0 +1,130 @@
+
+//DEFINITIONS
+var bullet = "- ";
+
+var button = $("<button>").attr("type", "submit").text("submit");
+var input = $("<input>").attr("type", "string");
+var label = $("<label>").append("Add an item <br>").append(this.input);
+
+var icon = function(src) {
```

Comment:
actually you can just remove `src` and `alt` from `Icon` and just do `<img src="https://cdn3.iconfinder.com/data/icons/google-material-design-icons/48/ic_delete_48px-128.png" alt="delete item">`